### PR TITLE
[0.12.0] Count Dropped As Present

### PR DESCRIPTION
**Pull request by pixelpxed**
| localhost_3000_en-US_search_students_results_full_name=Chanachon(iPad Mini) | localhost_3000_en-US_search_students_results_full_name=Chanachon(iPad Mini) |
| --- | --- |

**Features**
- Count dropped as `present`, moved from `onLeave` 
  Resolves FRO-205 [(link)](https://linear.app/skiso/issue/FRO-205/attendance-count-dropped-as-present)
- Hide teachers' birthday from students
  Resolves FRO-207 [(link)](https://linear.app/skiso/issue/FRO-207/hide-teachers-birthday-from-students)

**Fixes**
- Absence type 'history.item.null' has no fallback translation
- 'อาหารที่แพ้' is misspelled in Student List Printout (Duplicate)
- 'อาหารที่แพ้' is misspelled in About
  Fixes FRO-196 [(link)](https://linear.app/skiso/issue/FRO-196/%E0%B8%AD%E0%B8%B2%E0%B8%AB%E0%B8%B2%E0%B8%A3%E0%B8%97%E0%B8%B5%E0%B9%88%E0%B9%81%E0%B8%9F%E0%B9%89)

### CODE REVIEW

File: translations/search/students/absenceHistoryDialog/en-US.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "history": {
     "title": "Absence history",
     "item": {
+      "null": "Unknown reason",
```
Comment (SiravitPhokeed) [inline translations/search/students/absenceHistoryDialog/en-US.json:5]: I think “No data” would be more appropriate. Pretty sure that’s the phrase used elsewhere in the app as well.

**Review by SiravitPhokeed — APPROVED** [review]
LGTM!

File: translations/search/students/absenceHistoryDialog/en-US.json (hunk 1 above)
Comment (pixelpxed) [reply translations/search/students/absenceHistoryDialog/en-US.json:5]: Good idea.

File: translations/search/students/absenceHistoryDialog/th.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "history": {
     "title": "ประวัติการขาดเรียน",
     "item": {
+      "null": "ไม่มีข้อมูล",
```
Comment (CursorWind) [inline translations/search/students/absenceHistoryDialog/th.json:5]: Good translation choice!

**Review by CursorWind — APPROVED** [review]
LGTM

**Review by Sun-52 — APPROVED** [review]
LGTM!

**Pull request merged**